airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/EnabledControl.tsx

```diff
@@ -4,10 +4,11 @@ import styled from "styled-components";
 
 import { Toggle } from "components";
 
-import { Connection, ConnectionStatus } from "core/domain/connection";
 import { useAnalyticsService } from "hooks/services/Analytics/useAnalyticsService";
 import { useUpdateConnection } from "hooks/services/useConnectionHook";
 
+import { ConnectionStatus, WebBackendConnectionRead } from "../../../../../core/request/GeneratedApi";
```

Comment:
I noticed a few of these imports are `../../../core/request` but we have aliases in the codebase to just start with `core/request...`

Reply:
Discussion resulted in https://github.com/airbytehq/airbyte/pull/12319